packages/parameters/tests/helpers/cdkAspectGrantAccess.ts

```diff
@@ -35,7 +36,7 @@ export class ResourceAccessGranter implements IAspect {
     if (node instanceof NodejsFunction) {
 
       // Grant access to the resources
-      this.resources.forEach((resource: Table | Secret | StringParameter | IStringParameter) => {
+      this.resources.forEach((resource: Table | Secret | StringParameter | IStringParameter | CfnDeployment) => {
```

Comment:
This this gets uglier with every change. I am certain this PR is not the right time, but maybe we can at least drop a todo here. It needs a bit deeper investigation through all e2e tests to find a clean way to grant permissions to test functions.

Reply:
Good point.

I have created an issue to track this, we can address it after the beta release, but before the GA one (#1314).